app/views/command-line.html

```diff
@@ -33,19 +33,10 @@ <h1 id="cli">Command Line Tools</h1>
                   </p>
                   <p>
                     After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
-                    <div class="code prettyprint ng-binding" ng-if="sessionToken">
-                      oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="toggleShowSessionToken()" ng-show="!showSessionToken">...click to show token...</a>
-                    </div>
-                    <pre class="code prettyprint ng-binding" ng-if="!sessionToken">
-                      oc login {{loginBaseURL}}
-                    </pre>
+                    <pre class="code prettyprint">oc login {{loginBaseURL}} <span ng-if="sessionToken">--token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="toggleShowSessionToken()" ng-show="!showSessionToken">...click to show token...</a></span></pre>
                   </p>
 
-                  <div ng-show="showSessionToken" class="alert alert-warning">
-                    <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
-                    <strong>A token is a form of a password.</strong>
-                    Do not share your API token.
-                  </div>
+                  <token-warning show-session-token='showSessionToken'></token-warning>
```

Comment:
If this is just static content, I don't think we need a directive. I suggest just using ng-include here and in the about page.

```html
<div ng-if="showSessionToken" ng-include="'views/token-warning.html'"></div>
```

where token-warning.html has your markup.